pkg/interface/src/views/apps/permalinks/TranscludedNode.tsx

```diff
@@ -269,6 +269,21 @@ export function TranscludedNode(props: {
 }) {
   const { node, showOurContact, assoc, transcluded } = props;
   const group = useGroupForAssoc(assoc)!;
+
+  if (typeof node?.post === 'string') {
+    return (
+      <Box
+        mx="12px"
+        mt="12px"
+        p="2"
+        backgroundColor="washedGray"
+        borderRadius="2"
+      >
+        <Text gray>This message has been deleted.</Text>
```

Comment:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/20846414/117489303-75d8b600-af3b-11eb-90d8-7bf629d60459.png">

Reply:
Right, missed that it returns before it has a chance to evaluate the switch. Unfortunately these boxes have margins unto themselves that would look out of place in the ChatMessage context (since we use _that_ notice back in the message stream itself).  Just pushed a change.